sirepo/template/srw.py

```diff
@@ -946,9 +909,14 @@ def write_parameters(data, run_dir, is_parallel):
         run_dir (py.path): where to write
         is_parallel (bool): run in background?
     """
+    if data.report == 'exportRsOpt':
+        p = ''
+        _export_rsopt_config(data)
+    else:
+        p = _trim(_generate_parameters_file(data, run_dir=run_dir))
     pkio.write_text(
         run_dir.join(template_common.PARAMETERS_PYTHON_FILE),
-        _trim(_generate_parameters_file(data, run_dir=run_dir))
+        p
```

Comment:
add trailing comma

Reply:
Add a comma where? `pkio.write_text()` takes 2 arguments, or am I overlooking something?

Reply:
Our style is this:

```py
 pkio.write_text(
        run_dir.join(template_common.PARAMETERS_PYTHON_FILE),
        _trim(_generate_parameters_file(data, run_dir=run_dir))
        p,
)
```

Trailing commas on all lines even when they aren't necessary means that you can reorder or add/delete lines without knowing or disturbing the context of the other lines.

Reply:
I understand it for lists and the like, but for function args it seems misleading. Anyway, [f7b95c4](https://github.com/radiasoft/sirepo/pull/4057/commits/f7b95c4ecd35be76f5c3171076435f21698cb7a9)

Reply:
To finish my thought on this: to me `fn(a, b,)` looks like the function could take more parameters even if it can't. It blurs the meta-information. It is also visually ragged. I accept that this is a "me" problem.